README.md

```diff
@@ -19,7 +21,7 @@ This repository is fully supported by NSIDC. If you discover any problems or bug
 please submit an Issue. If you would like to contribute to this repository, you may fork
 the repository and submit a pull request.
 
-See the [LICENSE](LICENSE) for details on permissions and warranties. Please contact
+See the [LICENSE](LICENSE.md) for details on permissions and warranties. Please contact
```

Comment:
See note above. There is now both a`LICENSE` and `LICENSE.md` in this branch. :-)